ipn/ipnstate/ipnstate.go

```diff
@@ -81,6 +84,18 @@ type TailnetStatus struct {
 	MagicDNSEnabled bool
 }
 
+// ExitNodeStatus is the exit node used by the local backend.
+type ExitNodeStatus struct {
+	// Congigured is the exit node's ID.
```

Comment:
typo

Reply:
But how about just naming this field "ID"?

Then docs are:

// ID is the currently selected exit node.